### PR TITLE
kh360-react-622-evaluation-result-use-table-component

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Evaluation Results - Use table component
[#622](https://github.com/nerubia/kh360-react/issues/622)

### CODE REVIEW

File: src/features/evaluation-results/evaluation-results-list-table.tsx [hunk 1]
```diff
@@ -37,47 +39,43 @@ export const EvaluationResultsListTable = () => {
     navigate(`/evaluation-results/${id}`)
   }
 
+  const columns = [
```
Comment (gborgonia-ir) [inline src/features/evaluation-results/evaluation-results-list-table.tsx:42]: can pls transfer inside types file

Comment (chadiegilnerubia) [reply src/features/evaluation-results/evaluation-results-list-table.tsx:42]: :noted

File: __tests__/pages/evaluation-results/evaluation-results.spec.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (emmaacha) [inline __tests__/pages/evaluation-results/evaluation-results.spec.ts:1]: Please remove

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: __tests__/pages/evaluation-results/evaluation-results.spec.ts (hunk 2 above)
Comment (chadiegilnerubia) [reply __tests__/pages/evaluation-results/evaluation-results.spec.ts:1]: Noted

**Review by emmaacha — APPROVED** [review]

**Review by gborgonia-ir — APPROVED** [review]

**Pull request merged**